python/sdk/test/model_test.py

```diff
@@ -36,8 +36,14 @@
 responses = Responses('requests.packages.urllib3')
 
 default_resource_request = cl.ResourceRequest(1, 1, "100m", "128Mi")
+gpu_config = {"gpu_resource_type": "nvidia.com/gpu", "gpu_request": '1', "gpu_node_selector": {"cloud.google.com/gke-accelerator": "nvidia-tesla-p4"}}
+default_resource_request_with_gpu = cl.ResourceRequest(1, 1, "100m", "128Mi", **gpu_config)
```

Comment:
Same as above comment, we don't need to specify GPU in default_resource_request.